pages/ksphere/konvoy/1.5/install/install-aws/index.md

```diff
@@ -179,11 +181,11 @@ To customize the cluster name, run the following command:
 konvoy up --cluster-name <YOUR_SPECIFIED_NAME>
 ```
 
-**NOTE:** The cluster name may only contain the following characters: `a-z, 0-9, . - and _`.
+<p class="message--note"><strong>NOTE: </strong>The cluster name may only contain the following characters: `a-z, 0-9, . - and _`.</p>
 
 ## Show planned infrastructure changes
 
-Before running `konvoy up` or `konvoy provision` it is also possible to show the calculated changes that would be performed on the infrastructure by [Terraform][terraform].
+Before running `konvoy up` or `konvoy provision` it is also possible to show the calculated changes that would be performed on the infrastructure by <a href="https://www.terraform.io/" target="_blank">Terraform</a>.
```

Comment:
I thought we don't have to do this anymore?

Reply:
I'll fix it tonight

Reply:
The Konvoy PR https://github.com/mesosphere/konvoy/pull/1723 is not ready yet.
This would be fixed with the next release, or out of band if we want to.
Either way, these will work.